### ISSUE TITLE
Finalize 'Case to Give' webpage copy

### ISSUE THREAD

**Issue by KevZawacki** [MozillaFoundation/Advocacy#247]
Detailed description of work: Rework [current draft copy](https://docs.google.com/document/d/1YXEooaZfMO3jQoBTzgoLqWmVXQYKNnNkoUXjhiFJqtM/edit) into final form so we can move forward with design and implementation. (Aiming to launch on Oct. 1.)

Design scope happening in #248 

Due date: Sept. 2.

CC: @valianttry @HPaulJohnson

**Comment by KevZawacki** [comment]
@valianttry @HPaulJohnson 

Round 2 of our Case To Give page is ready for review. It includes:
- [Fresh copy](https://docs.google.com/document/d/1YXEooaZfMO3jQoBTzgoLqWmVXQYKNnNkoUXjhiFJqtM/edit#bookmark=id.yjf15k1o4ixe)
- [A handful more benchmarks](https://docs.google.com/document/d/1YXEooaZfMO3jQoBTzgoLqWmVXQYKNnNkoUXjhiFJqtM/edit#bookmark=id.ny4xl4xhufjs)
- [URL and localization considerations](https://docs.google.com/document/d/1YXEooaZfMO3jQoBTzgoLqWmVXQYKNnNkoUXjhiFJqtM/edit#bookmark=id.5auecvbm2qoc)

**Comment by damiankay** [comment]
Hi @KevZawacki - what's the status of this? Can we update the milestone? Thx!

**Comment by KevZawacki** [comment]
Hi @damiankay, now awaiting review from Andrea. Not urgent, so we can definitely push the milestone -- will do so now.

**Comment by valianttry** [comment]
@beccaklam

**Comment by KevZawacki** [comment]
Hi @valianttry, checking in -- anything I can do to help move this forward?

**Comment by valianttry** [comment]
hi @KevZawacki ! Wow the 2.0 is really a leap forward. Thank you for that great work! I added a few comments and such in the doc. Overall I would just say that we should make a donor the hero of our story (not the person who funds Mozilla, the hero). Does that make sense? I did a little "active voice" and tightening, but mostly left it. It's great.

@caltrider was asking how she might be able to help with the Fundraising campaign, and I thought about this project. She's a great storyteller and might be able to work together with you while I'm out next week to get this even closer to "done". I think it's close!
